src/metabase/driver.clj

```diff
@@ -398,6 +398,21 @@
     ;; Does the driver support percentile calculations (including median)
     :percentile-aggregations})
 
+(defmulti database-supports?
+  "Does this driver and specific instance of a database support a certain `feature`?
+  (A feature is a keyword, and can be any of the ones listed above in `driver-features`.
+  Note that it's the same set of `driver-features` with respect to
+  both database-supports? and supports?)
+
+  (supports? :mongo :set-timezone mongo-db) ; -> true"
+  {:arglists '([driver feature database])}
+  (fn [driver feature database]
+    (when-not (driver-features feature)
+      (throw (Exception. (tru "Invalid driver feature: {0}" feature))))
+    [(dispatch-on-initialized-driver driver) feature])
+  :hierarchy #'hierarchy)
+
+
 (defmulti supports?
   "Does this driver support a certain `feature`? (A feature is a keyword, and can be any of the ones listed above in
   `driver-features`.)
```

Comment:
Once you mark this as deprecated you should update the docstring explaining that people should _use_ `[[database-supports?]]` instead going forward, altho driver authors should continue _implementing_ `supports?` for the time being until we get a chance to migrate all of our usages